### DISCUSSION TITLE
delimiters is not working in my Django project.

### DISCUSSION THREAD

**Discussion by shahensaifullah** [vuejs/petite-vue#82] [Q&A]
![image](https://user-images.githubusercontent.com/44544213/133043407-41244555-775a-4248-9cda-445eb0e0a6a0.png)
delimiters are not working here. How can I use this in my Django project?

**Comment by misaacs21** [comment]
Are you using the delimiters in a v-for or v-if? Custom delimiters completely break within v-for and v-if loops (and presumably other v-*)

**Reply by halivert** [reply]
I've opened a pull request fixing this issue #90.

**Reply by shahensaifullah** [reply]
actually this issue is for the previous version of petite-vue. before 0.3, at that time, we couldn't use delimeter.

**Comment by niutech** [comment]
In your screenshot you are using delimiters `$[[` and `]]`, but you use `[[ count ]]` above. Where is `$`?

**Reply by shahensaifullah** [reply]
actually this issue is for the previous version of petite-vue. before 0.3, at that time, we couldn't use delimeter.
